test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1

```diff
@@ -13,10 +13,15 @@ Describe "Get-Process for admin" -Tags @('CI', 'RequireAdminOnWindows') {
         }
     }
 
-    It "Should support -FileVersionInfo" -Skip:(!$IsWindows) {
+    It "Should support -FileVersionInfo" {
         $pwshVersion = Get-Process -Id $pid -FileVersionInfo
         $PSVersionTable.PSVersion | Should -Match $pwshVersion.FileVersion
```

Comment:
This line should be changed to: $pwshVersion.FileVersion | Should -BeExactly $PSVersionTable.PSVersion

Reply:
Fixed.

Reply:
The versions are not exactly the same in the production for windows: we have "6.1.0-preview.8546" vs '6.1.0'.

Reply:
You'll have to strip the label so that they can match

Reply:
Do you mean to just take the substring from $PSVersionTable.PSVersion ?